src/View/Helper/FormCollection.php

```diff
@@ -170,7 +170,7 @@ public function render(ElementInterface $element): string
     /**
      * Only render a template
      */
-    public function renderTemplate(CollectionElement $collection): string
+    public function renderTemplate(CollectionInterface $collection): string
```

Comment:
This is a BC break for anyone who overrides this method, because this class is not final

Reply:
Ah OK. I thought because it was `@final` that would be OK.